src/test/scala/uk/gov/hmrc/test/ui/pages/section5/AdditionalProcedureCodesPage.scala

```diff
@@ -62,4 +62,6 @@ object AdditionalProcedureCodesPage extends BasePage {
   }
 
   def isLowValueDeclaration: Boolean = detail(AdditionalProcedureCodes(itemId)).contains(lowValueDeclaration)
+
+  def isNoOtherAdditionalProcedureCodeApplies: Boolean = detail(AdditionalProcedureCodes(itemId)) == "000"
```

Comment:
Nit: WDYT to rename this method to `hasZeroAdditionalProcedureCode`? 

`isNoOtherAdditionalProcedureCodeApplies` is not really meaningful.

Reply:
Done